src/components/NoData.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import LinkButton from 'components/LinkButton';
+
+import { ContainerCenter, Title } from 'styles/data-list';
+
+interface NoDataProps {
+  title: string;
+}
+
+const NoData = ({ title }: NoDataProps): JSX.Element => {
```

Comment:
Could please choose a more explicit name for this component? is it a page? A display? A table?